### DISCUSSION TITLE
IKEA E1812 TRÅDFRI Shortcut button - long press event name change?

### DISCUSSION THREAD

**Discussion by sopvkore** [EPMatt/awesome-ha-blueprints#511] [General]
I have just obtained an E1812 button (firmware version 0x23015631), ZHA integration. Not sure if it is a hw revision or sth else, but the long press event is named  "move_with_on_off_MoveMode.Up_83" while in the blueprint it is registered as ' "move_with_on_off_0_83'.
Moreover, the stop (aka. button release) event is named "stop_with_on_off" instead of just 'stop' as it is in the current blueprint.

It's clear what is the difference, so I have fixed my local blueprint version. But I'm not sure how can we make it universal. Perhaps two versions, perhaps a wildcard match (not sure if it even possible)? 
I can create an issue report and contribute the fix, but I'm just not sure what would be the right way to do it.

**Comment by scorfman** [comment]
I just picked a couple of these up and they also have firmware you mention. The behavior and fix you describe work great.
